db/1.10/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -626,9 +626,7 @@ This operation inserts data into a database.
 === Select
 `<db:select>`
 
-
-This operation selects data from a database. Streaming is automatically applied to avoid preemptive consumption of such results, which may lead to performance and memory issues.
-
+This operation queries data from a database. To prevent loading all the results at once, which could lead to performance and memory issues, results are automatically streamed. This means that pages of _fetchSize_ rows are loaded lazily when needed. If this operation is performed inside a transaction (e.g. within a try scope) and that transaction is closed before consuming the data, accessing the results which haven't been loaded will fail.
```

Comment:
```suggestion
This operation queries data from a database. To prevent loading all the results at once, which can lead to performance and memory issues, results are automatically streamed. This means that pages of `_fetchSize_ rows` are loaded when needed. If this operation is performed inside a transaction (that is, within a *Try Scope* component) and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```